### PR TITLE
Add example app and upgrade Flutter SVG to 0.19.1

### PR DESCRIPTION
**Pull request by dw2kim**

### CODE REVIEW

File: example/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
```
Comment (DiogoTheCoder) [inline example/test/widget_test.dart]: Could you either update this test file or remove it

**Review by DiogoTheCoder — CHANGES_REQUESTED** [review]
Could you also update the CHANGELOG and version number of this package

File: example/test/widget_test.dart (hunk 1 above)
Comment (dw2kim) [reply example/test/widget_test.dart]: Sure

**Review by DiogoTheCoder — APPROVED** [review]

**Comment by DiogoTheCoder** [comment]
Thanks for contributing, I've published the changes onto pub.dev, https://pub.dev/packages/height_slider

**Pull request merged**